jupyter/nmm-minimax.ipynb

```diff
@@ -131,7 +131,7 @@
     "$$ value(state, player, limit) = \\begin{cases}\n",
     "utility(state, player) & falls & finished(state, player) = true \\\\\n",
     "heuristic(state, player) & falls & limit = 0 \\\\\n",
-    "max(-value(ns, \\neg{player}, limit))  \\forall  ns \\in nextStates(state, player) & sonst\n",
+    "max( \\{-value(ns, opponent(player), limit) : ns \\in nextStates(state, player)\\} ) & sonst\n",
```

Comment:
Warum nicht mit dem `|` trennen (`\vert`)? So macht man das doch mathmatisch oder? Ich kenne die Schreibweise mit dem Doppelpunkt jedenfalls nicht ^^

![image](https://user-images.githubusercontent.com/52164046/123542949-9f001080-d74c-11eb-9d76-ec5524616987.png)

Und im Code haben wir keine Menge sondern eine Liste. Weiß nicht, ob wir das dann formell auch so machen sollten? Bzw ob man das im Code zu einer Menge macht, weil die ja glaube schneller als Listen sind in Python? :D

Reply:
Ich weiß, das ist eine schlechte Quelle, aber ich habe mir in Wikipedia Inspiration gesucht um mal zu schauen wie man sowas macht :D Da hatte ich das dann so gesehen, aber an `|` kann ich mich auch erinnern, jetzt wo du es sagst.

Ich denke, dass es am schnellsten wäre, wenn wir die List komplett weg lassen würden und einfach das resultierende Generator-Object verwenden würden.. Aber das Hilft uns bei der Beschreibung recht wenig. Ich bin mir nicht sicher, ob man die Schreibweise außerhalb von Mengen verwenden darf.